examples/devcontainer.json

```diff
@@ -1,4 +1,5 @@
 {
+  // Do not forget to remove all comments when you use this example file as JSON does not allow to have comments
```

Comment:
hm, maybe we should offload comments to a README.md instead?

Reply:
That could also work, I will make a suggestion

Reply:
I have moved the comments into a `README.md` just for the examples directory